core/templates/components/ck-editor-helpers/ck-editor-copy-content-service.ts

```diff
@@ -153,6 +158,15 @@ export class CkEditorCopyContentService {
     this.copyModeActive = !this.copyModeActive;
   }
 
+  // Function to check if the parent element or the current element is
+  // ANGULAR_HTML_BIND or not.
```

Comment:
use `/*...*/ ` for docstring in js, see other places for ref

Reply:
I'll change it.

Reply:
Actually, with the new changes, I don't think this will be relevant anymore. I'm removing this function since I couldn't reproduce the "cannot find property tagName of null" error - something which this code wanted to fix.